src/nl/surf/eduhub_rio_mapper/ooapi/course.clj

```diff
@@ -50,9 +50,15 @@
 
 ;; must have at least one rio consumer
 (s/def ::Course/consumers
-  (s/cat :head (s/* ::common/consumer)
-         :rio ::Course/rio-consumer
-         :tail (s/* ::common/consumer)))
+  (s/with-gen
+    (s/and
+      not-empty
```

Comment:
Zet hier ajb een comment bij dat die `not-empty` er alleen in zit om foutmelding te verbeteren (en daarom ook de with-gen)

Ik ben nog niet zeker dat dit in de praktijk ook het probleem is dat de gebruikers nu hebben. Wat is de foutmelding als ze wel een valide consumer hebben maar geen validate rio-consumer?

Reply:
`() - Insufficient input in [:consumers] at [:consumers :rio]`. Dat wordt met deze PR `[] - failed: not-empty spec: :nl.surf.eduhub-rio-mapper.ooapi.Program/consumers` - beter, maar zeker niet goed. Dat zijn de beperkingen van de spec explain output. 

Als ze wel een valide consumer hebben maar geen valide rio-consumer krijg je de oude error, maar aan de `()` in de error zie je dat ze helemaal geen consumers hebben.

Ik zal de comment toevoegen.